non-critical-infra/modules/mailserver/default.nix

```diff
@@ -41,6 +47,26 @@
     path = "${config.mailserver.dkimKeyDirectory}/nixos.org.mail.key";
   };
 
+  sops.secrets."nixos.org.${arc.selector}.key" = {
```

Comment:
Should we reload `rspamd.service` when this changes? Please include [`restartUnits`](https://github.com/Mic92/sops-nix?tab=readme-ov-file#restartingreloading-systemd-units-on-secret-change) if appropriate.

Reply:
Looks similar to my: https://github.com/Mic92/dotfiles/blob/main/machines/eve/modules/rspamd/rspamd.nix